object-cache.php

```diff
@@ -1000,46 +1044,60 @@ protected function _connect_redis() {
 			}
 		}
 
-		$this->redis = new Redis;
-
 		if ( file_exists( $redis_server['host'] ) && 'socket' === filetype( $redis_server['host'] ) ) { //unix socket connection
 			//port must be null or socket won't connect
 			$port = null;
 		} else { //tcp connection
 			$port = ! empty( $redis_server['port'] ) ? $redis_server['port'] : 6379;
 		}
-		$this->redis->connect( $redis_server['host'], $port, 1, null, 100 ); // 1s timeout, 100ms delay between reconnections
-		$keys_methods = array(
-			'auth'     => 'auth',
-			'database' => 'select',
+
+		$defaults = array(
+			'host' => $redis_server['host'],
+			'port' => $port,
+			'timeout' => 1000, // TODO I multiplied this by 1000 so we'd have a common measure of ms instead of s and ms, need to make sure this gets divided by 1000
```

Comment:
Nope! It's resolved here: https://github.com/pantheon-systems/wp-redis/pull/196/files#diff-5a2b681f0d32956de83215ea32388177R1076